test/mjsonwp/mjsonwp-specs.js

```diff
@@ -217,18 +217,32 @@ describe('MJSONWP', async () => {
         json: {id: 'baz', sessionId: 'lol', value: ['a']}
       });
 
+      // make sure adding the optional 'id' doesn't clobber a route where we
+      // have an actual required 'id'
       await request({
-        url: 'http://localhost:8181/wd/hub/session/foo/element/bar/value',
+        url: 'http://localhost:8181/wd/hub/session/foo/frame',
         method: 'POST',
         json: {id: 'baz'}
+      });
+    });
+
+    it('should properly handle variable payload for set value call', async () => {
+      await request({
+        url: 'http://localhost:8181/wd/hub/session/foo/element/bar/value',
+        method: 'POST',
+        json: {sessionId: 'lol'}
       }).should.eventually.be.rejectedWith("400");
 
-      // make sure adding the optional 'id' doesn't clobber a route where we
-      // have an actual required 'id'
       await request({
-        url: 'http://localhost:8181/wd/hub/session/foo/frame',
+        url: 'http://localhost:8181/wd/hub/session/foo/element/bar/value',
         method: 'POST',
-        json: {id: 'baz'}
+        json: {sessionId: 'lol', value: ['a'], text: 'bla'}
+      });
+
+      await request({
+        url: 'http://localhost:8181/wd/hub/session/foo/element/bar/value',
+        method: 'POST',
+        json: {sessionId: 'lol', text: 'bla'}
```

Comment:
yes, and my understanding of this text file is that the particular driver (FakeDriver) is supposed to simply return whatever is passed into it. the fact that it's returning nothing makes me curious whether the value is getting passed into it to begin with (and that _is_ the job of the MJSONWP code)